packages/devtools/README.md

```diff
@@ -1,6 +1,6 @@
 # devtools
 
-Devtoolsd used in magmo projects
+Developer tooling, used by statechannels packages.
 
 ## Usage
 
```

Comment:
This readme points to the devtools under `github.com/magmo`, as well as to `@statechannels/devtools`. Are either of these correct?